runtime/binding-http-filesystem/src/main/java/io/aklivity/zilla/runtime/binding/http/filesystem/internal/config/HttpFileSystemBindingConfig.java

```diff
@@ -53,10 +47,12 @@ public HttpFileSystemRouteConfig resolve(
         long authorization,
         HttpBeginExFW beginEx)
     {
-        HttpHeaderFW pathHeader = beginEx != null ? beginEx.headers().matchFirst(HEADER_NAME_PATH) : null;
+        HttpHeaderFW methodHeader = beginEx.headers().matchFirst(h -> HEADER_METHOD_NAME.equals(h.name()));
+        String method = methodHeader != null ? methodHeader.value().asString() : null;
+        HttpHeaderFW pathHeader = beginEx.headers().matchFirst(h -> HEADER_NAME_PATH.equals(h.name()));
```

Comment:
Confirming that we don't need to protect against null beginEx here, right?

Reply:
added null check